docs/api/ConnectionClose.md

```diff
@@ -25,11 +25,15 @@ The valid handle to an open connection object.
 
 `ConnectionClose` cleans up and frees all resources allocated for the connection in `ConnectionOpen`.
 
-A caller should shutdown an active connection via [ConnectionShutdown](ConnectionShutdown.md) before calling `ConnectionClose`; calling `ConnectionClose` without [ConnectionShutdown](ConnectionShutdown.md) will implicitly call [ConnectionShutdown](ConnectionShutdown.md) with the `QUIC_CONNECTION_SHUTDOWN_FLAG_SILENT` flag.
+A caller should shutdown an active connection via [ConnectionShutdown](ConnectionShutdown.md) before calling
+`ConnectionClose`. Calling `ConnectionClose` without [ConnectionShutdown](ConnectionShutdown.md) will abortively and
```

Comment:
Is it the associated streams that are shutdown abortively and connection shutdown silently? If so, I think we should mention it explicitly.

Reply:
No, the connection is closed abortively and silently (as in, abortively != graceful, and silently = we don't even send an on-the-wire message to the peer about it).

Basically, if you call `ConnectionClose` without calling `ConnectionShutdown` first, we do a `ConnectionShutdown(QUIC_CONNECTION_SHUTDOWN_FLAG_SILENT)` on behalf of the app.

I improved the wording a tiny bit in the next iteration, but if that still feel unclear, I'm open to a suggestion for something better.